<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Integracion</string>
  </property>
  <widget class="QLineEdit" name="valorA">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="valorB">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Valor de b:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ecuacion">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ecuación:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Valor de a:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_trapecios">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Método trapecios</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_simpson1_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>470</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Método simpson 1/3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_simpson3_8">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>470</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Método simpson 3/8</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="mitabla">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>781</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>255</number>
   </attribute>
   <column>
    <property name="text">
     <string>Valor real</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Aproximación</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Error</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>120</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Puntos:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numero_puntos">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="cuadratura_gauss">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>570</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cuadratura de Gauss</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
